toolkit/scripts/toolchain/cross/gcc_cross.sh

```diff
@@ -42,52 +47,53 @@ cd gcc-9.1.0
 ln -s ../mpfr-4.0.1 mpfr
 ln -s ../gmp-6.1.2 gmp
 ln -s ../mpc-1.1.0 mpc
-
-# Patch GCC to avoid an error where PATH_MAX isn't defined
-patch -p1 < "${scriptDir}/pathmax.patch"
 cd ..
 
-sudo mkdir -p ${installDir}
+sudo mkdir -p "${installDir}/aarch64-linux-gnu"
```

Comment:
Is this now a no-op with `mkdir -p ${sysroot}` from line 30 above?

Reply:
Yup good catch. I removed the line